File: BACKEND-CORE/Core/Model/AppUserAuth.cs
using System.Collections.Generic;

namespace backend.Core.Model
{
  public class AppUserAuth
  {
    public AppUserAuth() : base()
    {
      UserName = "Not authorized";
      BearerToken = string.Empty;
    }

    public string UserName { get; set; }
    public string BearerToken { get; set; }
    public bool IsAuthenticated { get; set; }

    public List<AppUserRole> Roles { get; set; }
  }
}

File: BACKEND-JAVA/src/main/java/backend/model/AppUserAuth.java
package backend.model;

public class AppUserAuth
{
  private String _UserName;
  private String _BearerToken;
  private boolean _IsAuthenticated;

  public AppUserAuth()
  {
    _UserName = "Not authorized";
    _BearerToken = "";
  }

  public void SetUserName(String username){
    this._UserName = username;
  }

  public String UserName(){
    return this._UserName;
  }

  public void SetBearerToken(String bearerToken){
    this._BearerToken = bearerToken;
  }

  public String BearerToken(){
    return this._BearerToken;
  }

  public void SetIsAuthenticated(Boolean isAuthenticated){
    this._IsAuthenticated = isAuthenticated;
  }

  public boolean IsAuthenticated(){
    return this._IsAuthenticated;
  }


  //public List<AppUserRole> Roles { get; set; }
}